### PR TITLE
Test Pluribus too

### PR DESCRIPTION
**Pull request by mirceaulinic**

### CODE REVIEW

File: test/unit/TestPluribusDriver.py [hunk 1]
```diff
@@ -32,7 +32,7 @@ def setUpClass(cls):
         cls.device.open()
 
 
-class TestGetterPluribusriver(unittest.TestCase, TestGettersNetworkDriver):
```
Comment (mirceaulinic) [inline test/unit/TestPluribusDriver.py]: It's a driver not a river tho...

Comment (dbarrosop) [inline test/unit/TestPluribusDriver.py]: :)

**Comment by dbarrosop** [comment]
Nice! Looks like some test is failing, could you check?

**Comment by fooelisa** [comment]
+1 :)

**Pull request merged**